docs/stdlib/datetime.rst

```diff
@@ -827,54 +827,52 @@ EdgeDB stores and outputs timezone-aware values in UTC.
                   std::duration_get(dt: cal::date_duration, \
                                     el: str) -> float64
 
-    Extract a specific element of input duration by name.
+    Return a specific element of input duration given a unit name.
```

Comment:
This might lead to a merge conflict if accepted before #4464, is L830s current writing in that OR okay, or should I bring this over to that one?

Reply:
Thanks for calling this out. There are elements I like in each of them. What about a hybrid of the two?

>Returns the element of a duration given a unit name.

I'll go ahead and update on my side.